src/main/kotlin/com/noumenadigital/npl/cli/service/NplCommandsParser.kt

```diff
@@ -0,0 +1,16 @@
+package com.noumenadigital.npl.cli.service
+
+import com.noumenadigital.npl.cli.commands.NplCliCommandsRegistry
+import com.noumenadigital.npl.cli.commands.registry.NplCommandExecutor
+import com.noumenadigital.npl.cli.exception.CommandParsingException
+
+object NplCommandsParser {
+    fun parse(command: List<String>): NplCommandExecutor {
+        if (command.isEmpty()) {
+            throw CommandParsingException("No command to execute\n")
```

Comment:
Nit: I think it's the responsibility of the catch-site to format the message for printing.

```suggestion
            throw CommandParsingException("No command to execute")
```